base/src/model.rs

```diff
@@ -1219,6 +1226,17 @@ impl Model {
     /// * [Model::update_cell_with_bool()]
     /// * [Model::update_cell_with_formula()]
     pub fn update_cell_with_text(&mut self, sheet: u32, row: i32, column: i32, value: &str) {
+        // Checking first whether cell we are updating is part of Merged cells
+        // If so, returning with print message
+        // TODO : Any better error handling ? ==> Should we start returning Result<> op ?
```

Comment:
Correct, error handling is missing here. Maybe a first PR should tackle that? I'm not sure. What is the `sheet` doesn't exist? A public method should never panic.

Reply:
Here one doubt.
I see only **update_cell_with_formula** returns Result<(), String> where I can propogate the error back.

All the other updates does not return anything ( kept like that purposefully as its set functions ). If I change these in all of these public apis, I need to add error handling in all of the testcases as it starts to give warning ( saying we did not handle the error return values ).

Should I propagate the error back in these functions and modify the testcases ?

OR

Should I keep it as it is but just silently log it saying update did not go through ?

Reply:
Added basic error handling